spec/models/fee/basic_fee_type_spec.rb

```diff
@@ -2,12 +2,37 @@
 
 module Fee
   describe BasicFeeType do
+
     let(:fee_type)  { build :basic_fee_type }
+    DATES_ATTENDED_APPLICABLE_FEES = %w( BAF DAF DAH DAJ PCM SAF )
+    DATES_ATTENDED_NOT_APPLICABLE_FEES = %w( CAV NDR NOC PPE NPW )
```

Comment:
Why are these hard coded? Isn't there a risk that these dont match what is being used in actual code?